### DISCUSSION TITLE
osu!lazer crash after pressing Alt+TAB

### DISCUSSION THREAD

**Discussion by hmdnnrmn** [ppy/osu#15550] [Q&A]
osu!lazer crash after pressing Alt+TAB

**Comment by peppy** [comment]
logs https://github.com/ppy/osu/discussions/15188

**Reply by hmdnnrmn** [reply]
[network.log](https://github.com/ppy/osu/files/7509625/network.log)
[performance.log](https://github.com/ppy/osu/files/7509628/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7509629/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7509630/updater.log)

i encountered this problem in fullscreen only but in borderless this doesn't happend

**Reply by peppy** [reply]
No crash in logs, so going to be your graphics drivers.

**Comment by frenzibyte** [comment]
Sounds similar to https://github.com/ppy/osu/issues/12480.
